### PR TITLE
feat(Chip): Add ability to click Chips (#621)

### PR DESCRIPTION
**Pull request by Winde**
Requested through CMS-3021. Chips should now be clickable, not compatible with closable

### CODE REVIEW

**Review by vdedios — APPROVED** [review]

**Review by yceballost — APPROVED** [review]

File: src/chip.tsx [hunk 1]
```diff
@@ -76,24 +101,59 @@ const Chip: React.FC<ChipProps> = ({Icon, children, id, dataAttributes, active,
...
-                className={classnames(
-                    styles.chipVariants[active ? 'active' : overAlternative ? 'overAlternative' : 'default'],
-                    {
-                        [styles.chipInteractiveVariants[isDarkMode ? 'dark' : 'light']]: isInteractive,
-                    }
-                )}
-                paddingLeft={paddingLeft}
-                paddingRight={12}
-                {...getPrefixedDataAttributes(dataAttributes, 'Chip')}
+            <BaseTouchable trackingEvent={props.trackingEvent} onPress={props.onPress} as="a">
```
Comment (atabel) [inline src/chip.tsx:125]: why `as="a"` ?

File: src/chip.tsx [hunk 2]
```diff
@@ -76,24 +101,59 @@ const Chip: React.FC<ChipProps> = ({Icon, children, id, dataAttributes, active,
...
         );
     }
+
+    if (props.href) {
+        return (
+            <BaseTouchable
+                trackingEvent={props.trackingEvent}
+                href={props.href}
+                newTab={props.newTab}
+                dataAttributes={dataAttributes}
```
Comment (atabel) [inline src/chip.tsx:149]: why do you pass `dataAttributes` here? they are already applied in `renderContent`

**Review by elenagarcia5 — APPROVED** [review]

File: src/chip.tsx (hunk 2 above)
Comment (Winde) [reply src/chip.tsx:149]: will remove

File: src/chip.tsx (hunk 1 above)
Comment (Winde) [reply src/chip.tsx:125]: If I remove the as="a"

The styles change due to BaseTouchable:
image

Seems like button is styled

Comment (atabel) [reply src/chip.tsx:125]: hmmm, I see. I think I'd apply some styles to reset the button builtin styles, but I prefer this to be a `button`, not an `a`.
Something like this should work:
```js
sprinkles({
    border: 'none',
    background: 'transparent',
    padding: 0,
})
```

Comment (Winde) [reply src/chip.tsx]: applied css reset and reverted to button

**Pull request merged**